src/Postprocessor.py

```diff
@@ -121,6 +136,16 @@ def get_symbols_from_section(self, section_name: str):
                 symbols_in_section.append(symbol_table.getSymbols(addr)[0])
         return symbols_in_section
 
+    def detect_datatype_header_dependency(self, data_type: str):
+        header = libc_definitions.get(data_type)
+        if header is not None:
+            self.headers.add(header)
+
+    def detect_function_header_dependency(self, function_name: str):
+        header = types_from_libc.get(function_name)
+        if header is not None:
+            self.headers.add(header)
+
```

Comment:
First of all, need to swap dictionaries
`libc_definitions` should be in `detect_function_header_dependency` and `types_from_libc` in `detect_datatype_header_dependency`
Secondary, I think these functions are unnecessary. In `write_headers` we check value for None, so we can just replace their calls to 
`self.headers.add(types_from_libc.get(data_type))`
`self.headers.add(libc_definitions.get(f.getName()))`